frontend/src/components/Banner.tsx

```diff
@@ -0,0 +1,90 @@
+import { ReactNode } from "react";
+import { OutboundLink } from "react-ga";
+import { useLocalization } from "@fluent/react";
+import styles from "./Banner.module.scss";
+import warningIcon from "../../../static/images/icon-orange-info.svg";
+import { useLocalDismissal } from "../hooks/localDismissal";
+import { CloseIcon } from "./icons/close";
+import { useGaPing } from "../hooks/gaPing";
+
+export type BannerProps = {
+  children: ReactNode;
+  type?: "promo" | "warning";
+  title?: string;
+  illustration?: ReactNode;
+  cta?: {
+    target: string;
+    content: string;
+    onClick?: () => void;
+    gaPing?: Parameters<typeof useGaPing>[0];
+  };
+  dismissal?: {
+    key: string;
+    duration?: number;
+  },
+  hiddenWithAddon?: boolean;
+};
+
+export const Banner = (props: BannerProps) => {
+  const ctaRef = useGaPing(props.cta?.gaPing ?? null);
+  const dismissal = useLocalDismissal(props.dismissal?.key ?? "unused", { duration: props.dismissal?.duration });
+  const {l10n} = useLocalization();
+  const type = props.type ?? "warning";
+  const icon =
+    props.type === "warning" ? (
+      <img src={warningIcon.src} alt="" className={styles.icon} />
```

Comment:
No, I explicitly set it to the empty string, because it's just decoration:

> Setting this attribute to an empty string (alt="") indicates that this image is not a key part of the content (it’s decoration or a tracking pixel), and that non-visual browsers may omit it from [rendering](https://developer.mozilla.org/en-US/docs/Glossary/Rendering_engine).

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#attr-alt